### DISCUSSION TITLE
Is it possible to get contribution counts for a different time zone?

### DISCUSSION THREAD

**Discussion by DenverCoder1** [community/community#20898] [Other Feature Feedback, Questions, & Ideas]
I want to be able to get the number of contributions each day for a given user.

I currently have a query using the v4 GraphQL API that does this:

```gql
query {
    user(login: "$user") {
        contributionsCollection(from: "$start", to: "$end") {
            contributionCalendar {
                totalContributions
                weeks {
                    contributionDays {
                        contributionCount
                        date
                    }
                }
            }
        }
    }
}
```

The issue is that different users see their graphs differently depending on their timezone, for example, in India it would appear I did not miss a day on Saturday, July 16, 2022:
![image](https://user-images.githubusercontent.com/20955511/179846733-d8a9a667-67e7-4bfc-825b-42249f42a443.png)
But in United States, it would appear that I had 0 contributions that day:
![image](https://user-images.githubusercontent.com/20955511/179845542-87261e47-921d-4cf1-8fed-a6e2c5a47e7b.png)

https://github.blog/2014-03-07-timezone-aware-contribution-graphs/

The GraphQL API seems to be inconsistent with the results returned, sometimes counting contributions differently without any changes to the parameters or headers.

What I wonder is if it is possible using the GraphQL API or another method to get contribution counts exactly how they would appear for a user in any given timezone.

I have tried using the `Time-Zone` header as demonstrated [here](https://developer.github.com/changes/2014-03-04-timezone-handling-changes/) but it seems to have no effect on this endpoint.

```php
$headers = [
    "Authorization: bearer $token",
    "Content-Type: application/json",
    "Accept: application/vnd.github.v4.idl",
    "User-Agent: GitHub-Readme-Streak-Stats",
    "Time-Zone: Asia/Kolkata",  // <-- This seems to have no effect
];
```

**Comment by HD111995** [comment]
https://github.blog/2014-03-07-timezone-aware-contribution-graphs/

**Reply by DenverCoder1** [reply]
This doesn't answer the question. I already know this information and had even included it in the original post.
